apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,182 @@
+
+## Setup a Site Collection and Site  
+
+Now, that the deployment is finished, you will create your first site in XM Cloud.  
+
+1. You can access XM Cloud via the XM Cloud Deploy app through the “Go to XM Cloud” button 
+![XM Cloud Deploy Logs - Finished, Open XM Cloud Dashboard](https://sitecorecontenthub.stylelabs.cloud/api/public/content/51576832a2594fde8dd0fb7f86fbdfd1?v=59c586b8) 
+If you closed that screen already you can also access XM Cloud from the Sitecore Cloud Portal. Click the “Open app” button on the top right. 
+![Sitecore Cloud Portal - Open XM Cloud Dashboard](https://sitecorecontenthub.stylelabs.cloud/api/public/content/ef83f2ef7ff94688b9f84b59f57f98b5?v=8bed26f3) 
+2. Under the sites tab, click create site. 
+![Create new Site](https://sitecorecontenthub.stylelabs.cloud/api/public/content/08c2c56cb94c4ce8969c691a8b6ff4d3?v=c79d9844) 
+Here, you can choose to create an empty site, or a basic site that already has some content in it.  
+
+It is recommended to use the empty site as a starting point when you are creating your own websites outside the scope of this blog. The basic site is more for quick demo purposes, and would require excess cleanup when used for a client project.  However, for this blog, you will select to use the basic site.   
+![Choose Site Template](https://sitecorecontenthub.stylelabs.cloud/api/public/content/1480ca22c0724a7496c9617b5f65cd6a?v=cf3753f9) 
+
+3. Name the site `Company DEV`.  
+![Name new Site and Site Collection](https://sitecorecontenthub.stylelabs.cloud/api/public/content/1dc9460c15fd452681187485fae6fe37?v=8feccf45) 
+
+4. Select a pre-configured language, in this case, English.  
+
+At this time, you can also select an existing Site identifier, or create a new one. This is required when connecting CDP for embedded analytics and personalization.  
+
+Now, let’s start the creation process of the site.  
+
+5. Click `create website`, which will scaffold the website in the background. This will take a few minutes.  
+
+Once your site is scaffolded in the background, you can access it using XM Cloud Pages and start editing the content. 
+
+![Open Pages Editor](https://sitecorecontenthub.stylelabs.cloud/api/public/content/6438c1971da24b8f8564e3e48335d985?v=c5f57f17) 
+ 
+6. Click on the website, which opens up a WSYWIG editor where you can begin to edit the content, see components that you can use to edit the site, and switch between languages and your sites (top nav bar).   
+![In Pages Editor](https://sitecorecontenthub.stylelabs.cloud/api/public/content/9fb983d0eeec435797a1115f4ae801df?v=2cce57d4) 
+ 
+
+## Working locally  
+
+Let’s now navigate to your GitHub account and download the repository that was cloned earlier by the Sitecore XM Cloud Deploy App.  
```

Comment:
Instead of adding 'Let's now navigate...' I'd change this to: 

"Navigate to your GitHub account and download the repository that was cloned earlier by the Sitecore XM Cloud Deploy App."

Fewer words, and more to the point.

Reply:
This has been resolved.